app/jobs.js

```diff
@@ -320,6 +326,74 @@ var job = {
         }
 
 
+    },
+    getJob: async function(report_name){
+        try {
+            var exist_report = await models.Report.findOne({
+                include: [
+                    {
+                        model: models.ReportLineItem,
+                        as: 'reportline',
+                    },
+                    {
+                        model: models.AssignReport
+                    },
+                    {
+                        model: models.SchedulerTask,
+                        where:{ active: true }
+                    },
+                ],
+                where: {
+                    report_name:report_name.report_name
+                }
+            })
+            if ( exist_report ) {
+                return schedulerDTO(exist_report);
+            }
+            else {
+                var response = { message: "Reports Not Exist" }
+                return response;
+            }
+        }
+        catch (ex) {
+            var response = { success: 0, message: ex }
+            logger.log({
+                level: 'error',
+                message: 'error while fetching reports for user',
+                error: ex,
+              });
+            return response;
+        }
+
+
+    },
+    JobCountByUser: async function(userName){
+        try {
+            var reports = await models.Report.findAll({
```

Comment:
are you running query like select count(*) from table?we should not fetch all record